buildDependencies.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 
-npm --prefix core ci
-npm --prefix core run build
-npm --prefix bolt-connection ci
-npm --prefix bolt-connection run build
+cd core 
+npm ci
+npm run build 
+cd ../bolt-connection
```

Comment:
what about `npm --prefix core ...` and `npm --prefix ../bolt-connection`?

Reply:
The `--prefix` was generating packages not compatible with npm 6.x which is legacy but it still broadly used.